### ISSUE TITLE
Weekly quest - week 17

### ISSUE THREAD

**Issue by Unoacaso81** [ValveSoftware/Dota2-Gameplay#6105]
### Description

Good morning,
I clear all 5 stars for the quest: throw 150 candy into enemy wells, but my progress bar don't show any progress.

Can you help me?

### Screenshots

![immagine](https://user-images.githubusercontent.com/121255753/209205890-4bb4dd47-ca45-4ecc-bac7-95772948ec92.png)

**Comment by fry-** [comment]
I have the same issue. 

- I've finished 3 stars on the "Season's Greevilings" (top right) challenge, they were not counted.
- Then I finished 1 star on "Star Player" (top left), this star got counted.
- Afterwards, I finished another star on "Season's Greevilings" (top right), which then got counted.

This should be a server side issue so I'm not posting my computer specs. I'm on Linux (Lubuntu 20.04).

![20221223111645_1](https://user-images.githubusercontent.com/10758005/209319340-4accb402-81c2-418e-9f31-4ac7cea3ec30.jpg)
